tests/e2e/test_rosacli_cluster.go

```diff
@@ -584,36 +584,39 @@ var _ = Describe("Classic cluster creation validation",
 					Expect(stdout.String()).To(ContainSubstring(v))
 				}
 				if rosalCommand.CheckFlagExist("--multi-az") {
-					invalidReplicasErrorMapMultiAZ := map[string]string{
-						"2":  "Multi AZ cluster requires at least 3 compute nodes",
-						"0":  "Multi AZ cluster requires at least 3 compute nodes",
-						"-3": "must be non-negative",
-						"5":  "Multi AZ clusters require that the number of compute nodes be a multiple of 3",
-					}
-					for k, v := range invalidReplicasErrorMapMultiAZ {
-						rosalCommand.ReplaceFlagValue(map[string]string{
-							"--replicas": k,
-						})
-						stdout, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
-						Expect(err).NotTo(BeNil())
-						Expect(stdout.String()).To(ContainSubstring(v))
+					if !profile.ClusterConfig.AutoscalerEnabled {
```

Comment:
How about separating these testing logic about `if rosalCommand.CheckFlagExist("--multi-az") and else part` from this case, then the logic may be more clear? 

It may be divided into two cases when `profile.ClusterConfig.AutoscalerEnabled` is `true` or `not`, may use `skip`(like ln 2158~2160)  instead of nested if.

Just a suggestion.

Reply:
@86254860 Using nested if because we want to use one, or as least as possible, test case to cover on more cluster profile, at least keep a possibility to rich with more condition.